### PR TITLE
minor optimization on per response redirect prompt construction

### PR DESCRIPTION
**Pull request by gabemontero**
@openshift/devex fyi this was a sidebar optimization that came to me for the jenkins login redirect bump while investigating bugzilla 1483304 (actually link to bugzilla left out on purpose)

this change does not address the root cause of that bugzilla IMO (gory details in that bug)

### CODE REVIEW

**Comment by gabemontero** [comment]
[test]

File: src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftHttpRedirectWithPrompt.java [hunk 1]
```diff
@@ -35,37 +37,56 @@ public OpenShiftHttpRedirectWithPrompt(int statusCode, @Nonnull String url) {
 			throw new NullPointerException();
 		}
 		this.url = url;
+		this.redirect = new ArrayList<String>();
+        InputStream is = null;
```
Comment (bparees) [inline src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftHttpRedirectWithPrompt.java]: you really need to fix the tab/space settings on your editor :)

**Comment by gabemontero** [comment]
On Tue, Aug 22, 2017 at 1:04 PM, Ben Parees <notifications@github.com>
wrote:

> *@bparees* commented on this pull request.
> ------------------------------
>
> In src/main/java/org/openshift/jenkins/plugins/openshiftlogin/
> OpenShiftHttpRedirectWithPrompt.java
> <https://github.com/openshift/jenkins-openshift-login-plugin/pull/21#discussion_r134542270>
> :
>
> > @@ -35,37 +37,56 @@ public OpenShiftHttpRedirectWithPrompt(int statusCode, @Nonnull String url) {
>  			throw new NullPointerException();
>  		}
>  		this.url = url;
> +		this.redirect = new ArrayList<String>();
> +        InputStream is = null;
>
> you really need to fix the tab/space settings on your editor :)
>
lol ... i have insert spaces for tabs set, just double checked ... maybe I
just need to totally reformat all these files as I edit them,
and get rid of whatever funky tabs exist:

 [image: Inline image 1]

**Comment by gabemontero** [comment]
Ah ... my embedding of the screen capture did not work.  I'll try again:

screen

**Pull request merged**